messages/project.deploy.pipeline.quick.md

```diff
@@ -22,7 +22,7 @@ This command doesn't support source-tracking. The source you deploy overwrites t
 
 # flags.job-id.summary
 
-Job ID of the deployment you want to quick deploy.
+Job ID of the quick deployment.
```

Comment:
This is not the id of the quick deployment, it is the id of a previously validated deployment that now we want to quick deploy.